dist/scripts/vendor.js

```diff
@@ -8062,7 +8062,7 @@ delete c[b.id], delete d[a];
 http:80,
 https:443,
 ftp:21
-}, sf = b("$location"), tf = /^\s*[\\/]{2,}/, uf = {
+}, sf = b("$location"), tf = /^\s*[\\\/]{2,}/, uf = {
```

Comment:
That was the easiest way, but I decided to go the "right" way and downgraded from Node8. Did I miss somewhere in the README about <= Node4 being a requirement?